about.html

```diff
@@ -0,0 +1,201 @@
+              <li><a href="./index.html"> Home</a></li>
+              <li><a href="./index.html"> Courses</a></li>
+              <li><a id="joinicon" href="#"> Join</a></li>
+              <li><a id="sponsericon" href="#"> Sponser</a></li>
+              <li><a id="latesticon" href="#"> Latest</a></li>
+              <li>
+                <a href="#"><button class="nav-bar-btn">Register Now!</button></a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    </div>
+
+    <!--About Main home page section start from here -->
+    <section class="contianer-fluid about-page-fluid-container">
+      <!-- Mobile navbar start from here -->
+      <div class="container-fluid menu-bar-fluid">
+        <div class="row">
+          <div id="wrap" class="wrap">
+            <div id="bar1" class="menu"></div>
+            <div id="bar2" class="menu"></div>
+            <div id="bar3" class="menu"></div>
+          </div>
+        </div>
+
+        <div id="navlistdisplay2" class="row">
+          <ul class="nav-list-mobile">
+            <li><a href="./index.html"> Home</a></li>
+            <li>
+              <a id="logoiconabout" href="./about.html"> About Us</a>
+            </li>
+            <li>
+              <a id="pastbatchiconabout" href="#past-Batch-section">
+                Info</a
+              >
+            </li>
+            <li>
+              <a id="partnericonabout" href="#partner-section">Our Partners</a>
+            </li>
+            <li><a href="./index.html">Latest</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="container about-home-container">
+        <div class="row about-heading-1">
+          <h1>Welcome!</h1>
+        </div>
+        <div class="row about-heading-2">
+          <h2>
+            AFRICAN DANCE CONFERENCE 2023
+          </h2>
+        </div>
+        <div class="row about-para-heading">
+          <p>
+            Come enjoy a joyful harmonious infusion of cultural histories and stories fro more than 50 nations and more than 100 tribal commuNities and languages.
+          </p>
+        </div>
+        <div class="row about-heading-3">
+          <h3>
+            If you have any question about the conference, please feel free to contact us
+           
+          </h3>
+        </div>
+        <div class="row about-heading-4">
+          <h4>info@afrivandanceconference.com</h4>
+        </div>
+      </div>
+    </section>
+
+    
+    <!-- Past Batch section start from here -->
+    <section id="past-Batch-section" class="contianer-fluid past-section">
+      <div class="container">
+        <div class="row">
+          <div class="col-12 d-flex justify-content-center past-text">
+            See the Past conferences
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center">
+          <div class="col-1 past-text-line">line</div>
+        </div>
+        <div class="row past-para-heading">
+          <p>
```

Comment:
- [ ] There is a tricky thing here on the About page that you might have missed due to that the previous reviewer has not specifically pointed out. **The first one is that there is a partners section that is visible on the mobile version of the about page. Second is that the footer has the white background but also please make it big in order to match the design given as suggested by the previous reviewer**  Take a look at the comparison below 👇 

Given design
![image](https://user-images.githubusercontent.com/90222110/235261259-f8579818-6462-4045-8eb6-2847d65dd212.png)

Your design
![image](https://user-images.githubusercontent.com/90222110/235261523-fd6c5a6f-6604-4a97-9d59-00e6915cf4a9.png)